### DISCUSSION TITLE
Welcome to hugo-scroll Discussions!

### DISCUSSION THREAD

**Discussion by zjedi** [janraasch/hugo-scroll#115] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rafaelhaigermoser** [comment]
Hi,
thanks for the beautiful Theam! I use it at [religionsforfuture.at](https://religionsforfuture.at/).
But how does linking to an dedicated page work? Because they exist on this [demo page](https://zjedi.github.io/hugo-scroll/services/)  but when I cloned the example repo and started hugo server I got an 404 Page.
